CMakeLists.txt

```diff
@@ -79,6 +79,19 @@ SET(CMAKE_BUILD_TYPE "${CMAKE_BUILD_TYPE}" CACHE STRING
     "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel Maintainer."
     FORCE)
 
+# Explicit request to use CCACHE
+if($ENV{USE_CCACHE})
```

Comment:
Same here, I don't think forcing people to use ccache is nice. If someone wants to use it, we have a tutorial on the homepage.